crates/oxc_linter/src/rules/jest/valid_describe_callback.rs

```diff
@@ -0,0 +1,326 @@
+                ("Second argument must be a function", "Replace second argument with a function")
+            }
+            Self::NoAsyncDescribeCallback => {
+                ("No async describe callback", "Remove `async` keyword")
+            }
+            Self::UnexpectedDescribeArgument => (
+                "Unexpected argument(s) in describe callback",
+                "Remove argument(s) of describe callback",
+            ),
+            Self::UnexpectedReturnInDescribe => (
+                "Unexpected return statement in describe callback",
+                "Remove return statement in your describe callback",
+            ),
+        }
+    }
+}
+
+#[allow(clippy::too_many_lines)]
+#[test]
+fn test() {
+    use crate::tester::Tester;
+
+    let pass = vec![
+        ("describe.each([1, 2, 3])('%s', (a, b) => {});", None),
+        ("describe('foo', function() {})", None),
+        ("describe('foo', () => {})", None),
+        ("describe(`foo`, () => {})", None),
+        ("xdescribe('foo\', () => {})", None),
+        ("fdescribe('foo', () => {})", None),
+        ("describe.only('foo', () => {})", None),
+        ("describe.skip('foo', () => {})", None),
+        (
+            "
+            describe('foo', () => {
+                it('bar', () => {
+                    return Promise.resolve(42).then(value => {
+                        expect(value).toBe(42)
+                    })
+                })
+            })
+        ",
+            None,
+        ),
+        (
+            "
+            describe('foo', () => {
+                it('bar', async () => {
+                    expect(await Promise.resolve(42)).toBe(42)
+                })
+            })
+        ",
+            None,
+        ),
+        ("if (hasOwnProperty(obj, key)) {}", None),
+        (
+            "
+        describe.each`
+            foo  | foe
+            ${'1'} | ${'2'}
+        `('$something', ({ foo, foe }) => {});
+        ",
+            None,
+        ),
+    ];
+
+    let fail = vec![
+        ("describe.each()()", None),
+        ("describe['each']()()", None),
+        ("describe.each(() => {})()", None),
+        ("describe.each(() => {})('foo')", None),
+        ("describe.each()(() => {})", None),
+        ("describe['each']()(() => {})", None),
+        ("describe.each('foo')(() => {})", None),
+        ("describe.only.each('foo')(() => {})", None),
+        ("describe(() => {})", None),
+        ("describe('foo')", None),
+        ("describe('foo', 'foo2')", None),
+        ("describe()", None),
+        ("describe('foo', async () => {})", None),
+        ("describe('foo', async function () {})", None),
+        ("xdescribe('foo', async function () {})", None),
+        ("fdescribe('foo', async function () {})", None),
+        // TODO
+        // ("
+        //     import { fdescribe } from '@jest/globals';
+        //     fdescribe('foo', async function () {})
+        // ", None),
```

Comment:
How can I help?

Reply:
I want to know whether `fdescribe` is from import statement and its source is '@jest.globals'. I think #510 can help me.